src/test/java/donk/task/ToDoTest.java

```diff
@@ -0,0 +1,10 @@
+package donk.task;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+public class ToDoTest {
+    @Test
+    public void dummyTest() {
+        assertEquals(2, 2);
+    }
```

Comment:
Although this file might still be in progress, please don't forget to remove this unrelated dummy test when you are finished.